lib/cybersource/cyber_source_on_demand.rb

```diff
@@ -94,6 +94,10 @@ def empty?
           @response.params['merchantReferenceCode'].nil?
         end
 
+        def request_id
+          @response.params['requestID']
```

Comment:
Confused: the report is retrieved by querying by order_id and will contain all transactions for that payment (auth, capture(s), refund(s), etc.).

To which transaction will that `requestID` correspond?